Services/src/main/resources/webutils/templates/webutils-templates.html

```diff
@@ -1655,6 +1756,46 @@ <h4 class="modal-title">${attr('title')}</h4>
 			}, {"element": fileLink, "$scope" : context.$scope, "actionHelper" : context.actionHelper, "modelName": modelName}));
 		</script>
 	</attr-directive>
+	
+	<attr-directive name="ykOnDrag">
+		<script>
+			var funName = context.attributeValue;
+			
+			var element = context.element;
+			
+			$(element).off("dragstart").on("dragstart", $.proxy(function(event){
+				var methodName = eval("this.$scope." + this.funName);
+				
+			},{"$scope" : context.$scope, "funName" : funName}));
+		</script>
+	</attr-directive>
+	
+	<attr-directive name="ykOnDrop">
+		<script>
+			var funName = context.attributeValue;
+			
+			var element = context.element;
+			
+			$(element).off("drop").on("drop", $.proxy(function(event){
+				var methodName = eval("this.$scope." + this.funName);

```

Comment:
Same here also. eval result is not method name and it is not used also. Remove unwanted assignment